Project.toml

```diff
@@ -15,16 +17,20 @@ StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 TensorBoardLogger = "899adc3e-224a-11e9-021f-63837185c80f"
 
 [compat]
+Arrow = "2.2"
 CairoMakie = "0.7"
+Legolas = "0.3"
 Makie = "0.16.5"
 StatsBase = "0.33"
+Tables = "1.7"
 TensorBoardLogger = "0.1"
-julia = "1.5"
+julia = "1.6"
```

Comment:
Since we're making a breaking version bump, taking the opportunity to additionally bump the lowest supported Julia version.